Makefile

```diff
@@ -90,6 +93,22 @@ image: build_image
 	@mv -f deployments/image/configmap/Dockerfile.bkp deployments/image/configmap/Dockerfile
 	@rm -f ./deployments/image/configmap/${BINARY}
 
+.PHONY: admission
+admission:
```

Comment:
@sunilgovind: we should have standard targets to build all pieces for the repo. Are the **build** and **build_image** the ones that we are aiming for?
If they are we need a follow issue up to implement that here

Reply:
Yeah, a heads up to @sunilgovind , we might need a follow-up issue on that.